frontend/src/modules/vehicles/VehicleComments.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <div>
+    <CCard>
+      <CCardBody>
+        <CRow class="justify-content-between ml-0 mr-0">
+          <h4>Comments - ({{ commentsCount }})</h4>
+          <CButton color="primary" @click="writingComment = true" v-if="!writingComment">
+            <CIcon name="cil-pen"/>
+            &nbsp;&nbsp;Write a comment
+          </CButton>
+        </CRow>
+        <CCollapse :show="writingComment" :duration="400" class="mt-3">
+          <CCol>
+            <!-- <CTextarea
+              placeholder="New comment"
+              rows="5"
+              v-model="newComment"
+            /> -->
+            <quill-editor v-model="newComment" class="mb-3 mt-3"/>
+            <CButton color="danger" class="mr-2" @click="writingComment = false">Cancel</CButton>
+            <CButton color="success" @click="createNewComment">Post</CButton>
+          </CCol>
+        </CCollapse>
+        <hr />
+        <CCol v-if="comments.length > 0">
+          <div class="pt-2" v-for="(comment, index) in comments" :key="comment._id">
+            <CRow class="ml-0 mr-0 d-flex justify-content-between">
+              <h6>{{ comment.author.first_name }}&nbsp;{{ comment.author.last_name }}</h6>
+              <p><small>{{ comment.timestamp }}</small></p>
+            </CRow>
+            <div v-html="comment.comment">
```

Comment:
In line 31 in VehicleComments, there is a security hotspot as we are vulnerable to XSS attacks due to bypassing the auto-escaping Vue features by not having the v-html inside a child node.